### PR TITLE
update dockerfile and start.sh

### PR DESCRIPTION
**Pull request by weiqi-tori**

### CODE REVIEW

File: dis_001_significant_earthquakes/Dockerfile [hunk 1]
```diff
@@ -19,11 +19,11 @@ RUN mkdir -p /opt/$NAME/data
 WORKDIR /opt/$NAME/
 COPY contents/ .
 COPY time.cron .
-# VOLUME ./data
+VOLUME ./data
 
 # Set user for container security
 USER root
-RUN adduser -D $NAME
+RUN adduser $NAME
```
Comment (chrowe) [inline dis_001_significant_earthquakes/Dockerfile:26]: I guess `-D` just the current default values. Not sure why it was there to begin with.

File: ene_008_oil_chemical_spill_incidents/Dockerfile [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
+RUN pip install -U pip
 
 # install application libraries
-RUN pip install requests==2.22.0
+RUN pip install requests
 RUN pip install -e git+https://github.com/resource-watch/cartosql.py.git#egg=cartosql
 
 # env
-ARG NAME=ene_008
+ARG NAME=nrt-script
```
Comment (chrowe) [inline ene_008_oil_chemical_spill_incidents/Dockerfile:14]: Seems like some use `nrt-script` and others use the dataset id. Looks like this is just used as the name of the directory in Docker so seems fine to standardize on `nrt-script`.

File: ene_008_oil_chemical_spill_incidents/Dockerfile [hunk 3]
```diff
@@ -22,7 +21,7 @@ COPY contents/ .
 COPY time.cron .
 
 # Set user for container security
-RUN adduser -D $NAME
+RUN useradd $NAME
```
Comment (chrowe) [inline ene_008_oil_chemical_spill_incidents/Dockerfile:24]: Why use `useradd` here instead of `adduser`?

Comment (weiqi-tori) [reply ene_008_oil_chemical_spill_incidents/Dockerfile:24]: I copied that from another dataset when updating the docker file

Comment (chrowe) [reply ene_008_oil_chemical_spill_incidents/Dockerfile:24]: Probably doesn't matter but we are mostly using `useradd` so this helps us standardize.

File: ene_008_oil_chemical_spill_incidents/Dockerfile (hunk 2 above)
Comment (weiqi-tori) [reply ene_008_oil_chemical_spill_incidents/Dockerfile:14]: Updated in PR https://github.com/resource-watch/nrt-scripts/pull/1182

File: ene_008_oil_chemical_spill_incidents/Dockerfile (hunk 3 above)
Comment (weiqi-tori) [reply ene_008_oil_chemical_spill_incidents/Dockerfile:24]: Updated in PR https://github.com/resource-watch/nrt-scripts/pull/1183

**Pull request merged**